### ISSUE TITLE
Angle energy calculation bug

### ISSUE THREAD

**Issue by ajkluber** [ajkluber/model_builder#9]
The butane example shows that there is a problem in calculating the angle energy:
1. [Energy vs time](https://www.dropbox.com/s/t55qdjnxnt01jyh/Eterms_vs_time.pdf?dl=0)
2. [model_builder vs gromacs energy correlation](https://www.dropbox.com/s/mw3v6svwfdnk5an/Eterms_corr.pdf?dl=0)

Probably something got messed up in converting units.

**Comment by TensorDuck** [comment]
The fact that it doesn't look like a constant angle means there is probably something far worse than just a wrong constant somewhere. I'll go through the recent changes and see what it could've been. I'll let you know if I find anything.

**Comment by TensorDuck** [comment]
I believe the problem is fixed with my last commit. Two issues:
1. The example file defined the topology incorrectly: It defined everything in degrees, which messes up how the topol.top file writes.
2. The function calc_angle_energy in the Hamiltonian automatically converted angles to degrees. This of course won't work if everything is assumed to be radians.

I've also skimmed over other Hamiltonian methods to make sure there aren't any conversions we forgot, but I believe the issue is now fixed.
